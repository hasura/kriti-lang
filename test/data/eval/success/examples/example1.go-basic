{
  "guid": {{ $[0].guid }}
}