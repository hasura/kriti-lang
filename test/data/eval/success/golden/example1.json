{
    "guid": "43a922da-9665-4099-8dfc-f9af369695a4"
}